pkgs/os-specific/linux/iputils/default.nix

```diff
@@ -21,63 +22,26 @@ in stdenv.mkDerivation {
     owner = "iputils";
     repo = "iputils";
     rev = "s${time}";
-    sha256 = "19rpl48pjgmyqlm4h7sml5gy7yg4cxciadxcs24q1zj40c05jls0";
+    sha256 = "0b755gv3370c0rrphx14mrsqjb396zqnsm9lsws842a4k4zrqmvi";
   };
 
-  patches = [
-    (fetchpatch {
-      name = "dont-hardcode-the-location-of-xsltproc.patch";
-      url = "https://github.com/iputils/iputils/commit/d0ff83e87ea9064d9215a18e93076b85f0f9e828.patch";
-      sha256 = "05wrwf0bfmax69bsgzh3b40n7rvyzw097j8z5ix0xsg0kciygjvx";
-    })
-    (fetchpatch {
-      name = "add-missing-idn-declarations.patch";
-      url = "https://github.com/iputils/iputils/commit/5007d7067918fb3d950d34c01d059e5222db679a.patch";
-      sha256 = "0dhgxdhjcbb2q6snm3mjp38l066knykmrx4k8rn167cizn7akpdx";
-    })
-    (fetchpatch {
-      name = "fix-ping-idn.patch";
-      url = "https://github.com/iputils/iputils/commit/25899e849aa3abc1ad29ebf0b830262a859eaed5.patch";
-      sha256 = "1bqjcdjjnc2j6indcli7s7gbbhkcaligvh94asixfrmjzkbn533n";
-    })
-  ];
-
-  prePatch = ''
-    substituteInPlace doc/custom-man.xsl \
-      --replace "http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl" "${docbook_xsl}/xml/xsl/docbook/manpages/docbook.xsl"
-    for xmlFile in doc/*.xml; do
-      substituteInPlace $xmlFile \
-        --replace "http://www.oasis-open.org/docbook/xml/4.4/docbookx.dtd" "${docbook_xml_dtd_44}/xml/dtd/docbook/docbookx.dtd"
-    done
-  '';
+  # ninfod cannot be build with nettle yet:
+  patches = [ ./build-ninfod-with-openssl.patch ./tracepath-fix-build-with-musl.patch ]
+    # Avoid an unnecessary build time dependency on systemd:
+    ++ optional withSystemd [ ./use-systemd.patch ];
```

Comment:
Is this such an issue really, compared to maintaining a custom patch?

Reply:
Normally not, it only seemed inconvenient for cross-compiling, especially for projects which do not use `systemd` but I guess they would override it anyway. I agree that the custom patch isn't a good solution. Would you mind if I just dropped all unit files for now? I guess their use is limited, especially as long as we do not have a NixOS module for it.

I'd also be interested to hear your opinion on the Japanese translations (share/locale/ja/LC_MESSAGES/iputils.mo) - should we keep them or also ignore them for now?